### DISCUSSION TITLE
Cant run any packages in termux app on android

### DISCUSSION THREAD

**Discussion by A-AxeL** [termux/termux-packages#11904] [Q&A]
When I try to install NMap the install process go's fine, but when I want to use it, it gives me this:

"$ nmap
CANNOT LINK EXECUTABLE "nmap": library "libssl.so.3" not found: needed by main executable"

Same thing happens for ncat and many other programs.

Please help.

**Comment by tomty89** [comment]
Make sure you have openssl 3.0.5. If `pkg up` doesn't help, switch to another mirror with `termux-change-repo`.

**Reply by A-AxeL** [reply]
ok thx, i will try it. if it doesnt work, i will let u know

**Comment by sata0ata** [comment]
pkg up was help.

**Reply by trojanx6** [reply]
Hi, you can solve this error by updating the packages with the "pkg update" command and installing new ones.

**Comment by A-AxeL** [comment]
Thanks!
